Sources/Tracking/CustomerIO.swift

```diff
@@ -202,7 +202,18 @@ public class CustomerIO: CustomerIOInstance {
     private static func initialize(
         config: SdkConfig
     ) {
-        let newDiGraph = DIGraph(sdkConfig: config)
+        var sdkConfig = config
+
+        // temporary handling for backward compatibility with `autoTrackPushEvents`.
+        // if `autoTrackPushEvents` is false (disabled) and `autoTrackPushMetricEvents` is set to track all events (default),
```

Comment:
Shouldn't this be like this? 🤔 
```suggestion
        // if `autoTrackPushEvents` is true (enabled) and `autoTrackPushMetricEvents` is set to track all events (default),
```

Reply:
its more like, 

```
if `autoTrackPushEvents` = true
- `autoTrackPushMetricEvents` is enabled for both `delivered` and `opened`

if `autoTrackPushEvents` = false
- `autoTrackPushMetricEvents` is enabled for only `delivered`
```